portfolio/src/main/webapp/fetcher.js

```diff
@@ -0,0 +1,5 @@
+//quick fetch function 
+async function strFetch(){
+    servletText = await (await fetch('/strservlet')).text();
```

Comment:
"Declare all local variables with either `const` or `let`. Use `const` by default, unless a variable needs to be reassigned." - https://google.github.io/styleguide/jsguide.html#features-use-const-and-let